src/App.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import { v4 } from 'uuid';
+import './App.css';
+
+class App extends Component {
+  state = {
+    tempDescription: '',
+    todos: [
+      {
+        id: v4(),
+        description: 'delectus aut autem',
+        completed: false,
+      },
+      {
+        id: v4(),
+        description: 'quis ut nam facilis et officia qui',
+        completed: false,
+      },
+      {
+        id: v4(),
+        description: 'fugiat veniam minus',
+        completed: false,
+      },
+      {
+        id: v4(),
+        description: 'et porro tempora',
+        completed: true,
+      },
+    ],
+  };
+
+  handleSubmit = event => {
+    event.preventDefault(); //not sure why I had to add this but read about it online - it fixed an issue with the todo not appearing on the screen
```

Comment:
We discussed it in person, let me know if it's clear now.

Reply:
Yes - this is needed because I've used a form here and the default form action is to try and send this information somewhere, is that right?